lib/stego/bitPlane.ts

```
@@ -0,0 +1,158 @@
+        resultData[i + 2] = bitValue * 255;
+        resultData[i + 3] = a;
+        break;
+      case 'blue':
+        bitValue = (b >> bitPlane) & 1;
+        resultData[i] = bitValue * 255;
+        resultData[i + 1] = bitValue * 255;
+        resultData[i + 2] = bitValue * 255;
+        resultData[i + 3] = a;
+        break;
+      case 'combined':
+        const rBit = (r >> bitPlane) & 1;
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)